src/vectorsynchronizer.js

```diff
@@ -81,6 +81,22 @@ olcs.VectorSynchronizer.prototype.createSingleCounterpart = function(olLayer) {
     csPrimitives.show = olLayer.getVisible();
   });
 
+  var onOpacityChange = function() {
+    var bbs = csPrimitives.context.billboards;
+    var opacity = olLayer.getOpacity();
+    if (!goog.isDef(opacity)) {
+      opacity = 1;
+    }
+    bbs.olLayerOpacity = opacity;
+    var i, bb;
+    for (i = bbs.length - 1; i >= 0; --i) {
```

Comment:
Starting the iterator with `bbs.length` saves me from having to store it in a separate variable. This is a small optimization I use frequently when the order of iteration does not matter.